valuesets/ValueSet-UKCore-ConsentException.xml

```diff
@@ -0,0 +1,26 @@
+<ValueSet xmlns="http://hl7.org/fhir">
+  <id value="UKCore-ConsentException" />
+  <url value="https://fhir.hl7.org.uk/ValueSet/UKCore-ConsentException" />
+  <version value="0.0.1" />
+  <name value="UKCoreConsentException" />
+  <title value="UK Core ConsentException" />
+  <status value="draft" />
+  <date value="2022-11-28" />
+  <publisher value="HL7 UK" />
+  <contact>
+    <name value="HL7 UK" />
+    <telecom>
+      <system value="email" />
+      <value value="ukcore@hl7.org.uk" />
+      <use value="work" />
+      <rank value="1" />
+    </telecom>
+  </contact>
+  <description value="A set of codes that define a reason to trigger an exception to a base policy, consisting of the entirety of the SNOMED CT UK Coding System. This set of codes may not necessarily be complete or appropriate and may require further discussion in the future." />
```

Comment:
Can we have "An exception can be an addition or removal of access permissions." added to give a better understanding of this ValueSet. Words taken from HL7